------------------------------------------------------------
revno: 253
committer: Scott Moser <smoser@ubuntu.com>
branch nick: trunk
timestamp: Tue 2010-08-17 22:03:19 -0400
message:
  fix syntax error in util.py
=== modified file 'cloudinit/util.py'
--- old/cloudinit/util.py	2010-08-13 15:42:57 +0000
+++ new/cloudinit/util.py	2010-08-18 02:03:19 +0000
@@ -145,7 +145,7 @@
     if base.find("%s") >= 0:
         ud_url = base % ("user-data" + ext)
         md_url = base % ("meta-data" + ext)
-    else
+    else:
         ud_url = "%s%s%s" % (base, "user-data", ext)
         md_url = "%s%s%s" % (base, "meta-data", ext)
 

